downstream/modules/platform/proc-new-aap-instance-upgrade.adoc

```diff
@@ -0,0 +1,31 @@
+[id="proc-new-aap-instance-upgrade_{context}"]
+
+= Deploy a new instance for a side-by-side upgrade
+
+To proceed with the side-by-side upgrade process, deploy a second instance of Ansible Tower 3.8.x with the same instance groups configurations. This new instance will receive the content and configuration from your original instance, and will later be upgraded to {PlatformName} {PlatformVers}.
+
+== Deploy a new instance of Ansible Tower
+
+Start by downloading the Tower installer version that matches your original Tower instance version, then running the installer:
+
+. Download the Tower installer version that matches your original Tower instance by navigating to this link: https://releases.ansible.com/ansible-tower/setup/
+. Navigate to the installer, then open the `inventory` file using a text editor to configure the `inventory` file for a Tower installation:
+.. In addition to any Tower configurations, remove any fields containing `isolated_group` or `instance_group`.
++
+NOTE: For more information about installing Tower using the {PlatformNameShort} installer, see the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/index[{PlatformNameShort} Installation Guide] for your specific installation scenario.
+. Run the `setup.sh` script to begin the installation.
+
+With the new instance now installed, you will need to configure the Tower settings to match the instance groups from your original Tower instance.
```

Comment:
Suggest rewording to active voice: Once the new instance is installed, configur the Tower settings to match the instance groups from your original Tower instance.